src/v3/src/types/widget.ts

```diff
@@ -110,15 +110,14 @@ export type IDPDisplayType = 'PRIMARY' | 'SECONDARY';
 export type WidgetProps = Partial<WidgetOptions>;
 
 export type WidgetOptions = {
-  // // ui customizations
-  // renderers?: JsonFormsRendererRegistryEntry[];
-  // cells?: JsonFormsCellRendererRegistryEntry[];
-  // components?: Record<string, Component>;
+  // brand
+  brand?: Brand;
```

Comment:
This property needs work. Will update with better version

Reply:
As discussed, added a commit to move `designTokens` from top-level property into `theme.designTokens`